Question 20: Valid Parentheses

```diff
@@ -0,0 +1,26 @@
+class Solution {
+    public boolean isValid(String s) {
+        Stack<Character> stack = new Stack<Character>();
+        HashMap<Character, Character> brackets = new HashMap<Character, Character>();
+        brackets.put('{', '}');
+        brackets.put('(', ')');
+        brackets.put('[', ']');
+        for(int i = 0; i < s.length(); i++){
+            char current = s.charAt(i);
+            if(brackets.containsKey(current)){
+                stack.push(current);
+            }else if(brackets.containsValue(current) && !stack.empty()){
+                char popped = stack.pop();
+                if(brackets.get(popped) != current)
```

Comment:
Is this correct?

Anyway, make sure you always add comments as I listed in the early review. I will not review any further code unless it is commented, e.g. so I know whether the code passes tests or there are issues that need trouble shoot.